monix-execution/js/src/main/scala_3.0/monix/execution/atomic/AtomicNumber.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2014-2021 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.execution.atomic
+
+/** Represents an Atomic reference holding a number, providing helpers
+  * for easily incrementing and decrementing it.
+  *
+  * @tparam A should be something that's Numeric
+  */
+abstract class AtomicNumber[A] extends Atomic[A] {
+  /** Increment with the given integer */
+  def increment(v: Int = 1): Unit
+  /** Adds to the atomic number the given value. */
+  def add(v: A): Unit
+  /** Adds to the atomic number the given value. Alias for `add`. */
+  final def `-=`(value: A): Unit = subtract(value)
```

Comment:
We have abused macros in `Atomic` and it's time to take a step back. I don't think those aliases being defined as macros have any value. Macros help in those `transform` methods for inlining the given function parameter in what is a hot loop, otherwise the JVM is perfectly capable of inlining such method calls by itself.

That said maybe we should have feature parity and then get rid of those macros in 4.0.0.

So I'm going to revert this change, to have consistency.